x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/MlJobSnapshotUpgradeIT.java

```diff
@@ -137,8 +136,8 @@ private void testSnapshotUpgrade() throws Exception {
         assertThat(snapshots, hasSize(1));
         snapshot = snapshots.get(0);
         assertThat(snapshot.getLatestRecordTimeStamp(), equalTo(snapshots.get(0).getLatestRecordTimeStamp()));
-        // min version in upgraded 7.series is 7.9.0
-        assertThat(snapshot.getMinVersion(), equalTo(Version.V_7_9_0));
+        // min version in upgraded 7.series is 7.11.0
+        assertThat(snapshot.getMinVersion(), equalTo(Version.V_7_11_0));
```

Comment:
```suggestion
        assertTrue(snapshot.getMinVersion.after(UPGRADE_FROM_VERSION));
```

I'm not sure how this works, is 7.11 the min version for the rest of 7.x?

Reply:
Until the next change to the C++ code that introduces new sections of state that the older versions won't understand.

Reply:
>         assertTrue(snapshot.getMinVersion.after(UPGRADE_FROM_VERSION));

@davidkyle this won't work. Once we do a bwc test from 7.11->7.12 this assertion would fail :/

Reply:
@davidkyle @droberts195 what do you think of making it `assertTrue(snapshot.getMinVersion.onOrAfter(Version.V_7_11_0));` ?

This way we at least protect ourselves for the next time that c++ changes.

Reply:
I think the changes are infrequent enough to keep this more precise assertion. 

Can you extract a new constant with a more meaningful name instead of using V_7_11_0 please. Something like

`final Version CPP_COMPATIBILTIY_VERSION = Version.V_7_11_0`